plugins/inputs/postgresql_extensible/postgresql_extensible.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 	"time"
 
-	_ "github.com/jackc/pgx/stdlib"
+	_ "github.com/jackc/pgx/stdlib" //to register stdlib from PostgreSQL Driver and Toolkit
```

Comment:
I think we'll find this linter rather annoying.

Reply:
It is pretty valid rule, should be used for special cases only - I needed to add a comment only in 7 places.